src/artemis/external_interaction/zocalo/zocalo_interaction.py

```diff
@@ -76,7 +76,6 @@ def run_end(self, data_collection_id: int):
         self._send_to_zocalo(
             {
                 "event": "end",
-                "ispyb_wait_for_runstatus": "1",
```

Comment:
It's probably worth adding in a comment (at least for now) to indicate that this is different behaviour compared to non-artemis MX data collections.

Reply:
Sorry, I didn't mean that to sound as confrontational as it did